Resources/Prototypes/_White/Entities/Clothing/Shoes/modsuits.yml

```diff
@@ -0,0 +1,30 @@
+﻿- type: entity
+  parent: ClothingShoesBase
+  id: ClothingModsuitBootsInfiltrator
+  name: infiltrator modsuit boots
+  description: A special modular suit boots designed for compact folding inside modular suit control.
```

Comment:
"Purpose-built boots that can be neatly folded and store in the infiltrator suit's main unit."